pages/index.tsx

```diff
@@ -18,8 +18,10 @@ const Home = ({ communities }: HomeProps): JSX.Element => {
       <Header />
       <Content>
         <Image src="/logo.png" alt="Logo" width={500} height={500} />
-        <Trending communities={communities} />
-        <Navigation href="/addCommunity">Add community</Navigation>
+        <div>
```

Comment:
This div is fine in this case, but keep in mind that in the future divs will be super rare because there will be paddings etc that need to be adjusted and it will be a styled component in it's place.